.github/workflows/ci.yml

```diff
@@ -268,7 +268,7 @@ jobs:
       fail-fast: false
       matrix:
         # test the oldest supported version and main
-        typing-extensions-version: ['4.5.0', 'main']
+        typing-extensions-version: ['4.6.1', 'main']
```

Comment:
4.6.2 is out. Should we use that one?

Reply:
No this should be the minimum supported version. We don't need 4.6.2 as far as I can tell so let's stick to 4.6.1 for now.